include/picongpu/plugins/transitionRadiation/frequencies/LogFrequencies.hpp

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2013-2019 Heiko Burau, Rene Widera, Richard Pausch, Finn-Ole Carstens
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+
+namespace picongpu
+{
+namespace plugins
+{
+namespace transitionRadiation
```

Comment:
If this is the same code as in the radiation plugin, please just use that code directly.

Reply:
It is not too trivial to use the same frequency functors for the `radiation plugin` as well as the `transition radiation plugin`. E.g. `radiation_log_freq.hpp` uses `plugins::radiation::frequencies::omega_min` directly, because it is in the same namespace, whereas my `LogFrequencies.hpp` uses `plugins::transitionRadiation::frequencies::omegaMin` from the `transitionRadiation` namespace. I see two possible solutions for this problem:

- One merges the namespaces together to use the same functors. But this would deny the usage of both plugins in the same simulation. 
- On the other hand one could refactor the functors, so that they take omega_min, omega_max and N_omega as parameters, which requires refactoring in both plugins, but it seems like the clean way to me.

Reply:
Okay - that is a valid reason to keep is separate for now - or I do a quick refactoring today and we will tray to include an static object as template argument to give omega_min and omega_max.